Documentation/CodingGuidelines/PhpFileFormatting/FileStructure.rst

```diff
@@ -31,11 +31,11 @@ Namespace
 The namespace declaration of each PHP file in the TYPO3 Core shows
 where the file belongs inside TYPO3 CMS. The namespace starts with
 :code:`"TYPO3\CMS\"`, then the extension name in UpperCamelCase, a
-backslash and then the name of the subfolder of :code:`Classes/`, in
+backslash and then the name of the subfolder of :file:`Classes/`, in
 which the file is located (if any). E.g. the file
-:code:`typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php`
+:file:`typo3/sysext/frontend/Classes/ContentObject/ContentObjectRenderer.php`
 with the class :code:`ContentObjectRenderer` is in the namespace
-":code:`TYPO3\CMS\Frontend\ContentObject`".
+":file:`TYPO3\CMS\Frontend\ContentObject`".
```

Comment:
This is not a file, it's php code, so `core` or `php` would be right here.

The change above to `file` was right.